### PR TITLE
updated: riot indexing

### PR DESCRIPTION
**Pull request by GianlucaGuarini**
# Pull request motivation(s)

The Riot.js website was redesigned

### What is the current behaviour?

The old site content are still indexed in the search point to 404 sites

### What is the expected behaviour?

The new site content should be indexed

### CODE REVIEW

File: configs/riotjs.json [hunk 1]
```diff
@@ -1,31 +1,27 @@
...
+    "lvl3": "main h3",
+    "lvl4": "main h4",
+    "lvl5": "main h5",
+    "text": "main p, main li"
   },
   "conversation_id": [
     "580098926"
   ],
   "selectors_exclude": [
-    "#toc"
```
Comment (s-pace) [inline configs/riotjs.json]: Why did you remove that?

Comment (GianlucaGuarini) [reply configs/riotjs.json]: because that id doesn't exist anymore

**Comment by s-pace** [comment]
Deployed ✅

**Comment by GianlucaGuarini** [comment]
@s-pace thank you
